[ERROR] 2022-04-28 17:45:29 [product_detail] HTTPConnectionPool(host='www.syntop-ien.com', port=80): Max retries exceeded with url: /Products-1659294.html (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001287DC8A250>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。')) main.py:102
